### ISSUE TITLE
Publish unreleased changes

### ISSUE THREAD

**Issue by aduth** [fnando/browser#541]
## Description

There are a couple changes slated for release ([see CHANGELOG.md](https://github.com/fnando/browser/blob/main/CHANGELOG.md)) which have been merged for multiple years. We particularly have an interest in using the new `chromium_based?` method. Would you please consider publishing a new release containing these changes?

## How to reproduce

```
require 'browser'
Browser.new('some user agent').chromium_based?
```

## What do you expect

```
false
```

## What happened instead

```
undefined method `chromium_based?' for #<Browser::Unknown ...> (NoMethodError)
```

## Software:

- Browser gem version: 5.3.1

**Comment by aduth** [comment]
I think a workaround given the current implementation is:

```
browser.chrome? || browser.edge?
```

On that note, there may be [other browsers that should also be considered as Chromium based](https://en.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium)? Being able to use the `chromium_based?` abstraction would be preferable to checking individual browsers to allow for changes over time.

**Comment by fnando** [comment]
just released v6.0.0.

**Comment by aduth** [comment]
Thank you for releasing the new version! And for maintaining this useful gem ❤️
